omeapi/location/v1beta1/location.proto

```diff
@@ -15,7 +15,7 @@ message UpdateLocationRequest {
   string worker_id = 1;
   double lon = 2;
   double lat = 3;
-  google.protobuf.Timestamp update_time = 4;
+  int64 timestamp = 4;
```

Comment:
Using int64 seems to sacrifice clarity / strong typing. The client won't know if it is seconds or milliseconds.

Reply:
will be returned it to google.protobuf.Timestamp